websocketOutportDriver/websocketOutportDriverNodePart.go

```diff
@@ -116,8 +122,8 @@ func (o *websocketOutportDriverNodePart) FinalizedBlock(headerHash []byte) error
 
 // Close will handle the closing of the outport driver web socket sender
 func (o *websocketOutportDriverNodePart) Close() error {
-	// TODO: close the web socket here
-	return nil
+	o.isClosed.Toggle(true)
+	return o.webSocketSender.Close()
```

Comment:
can we safely call the Close method on the `webSocketSender` instance?

Reply:
Yes, I think so